src/pages/open-source-for-non-developers.md

```diff
@@ -0,0 +1,28 @@
+# Open Source for Non-Developers
+
+## What is Open Source?
+
+Open source is software whose source code that can be freely viewed and redistributed.
+Unlike products that contain proprietary code, open source code can be viewed by developers worldwide.
+It can also allows people to propose changes, which fosters a collaborative working environment.
+
+## Why do Products Become Open Source?
+
+1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy, which in turn can save your company time and effort in creating troubleshooting guides.
```

Comment:
[Suggestion] It's more than just troubleshooting guides. It's that, and adding bugfixes & features, and etc.

```suggestion
1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy, which in turn can save your company time and effort.
```